From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@SilviaDoomra__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sdoomra). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rjagiewich committed:
Merge pull request #31468 from cwatson-cat/1-14-18NewButton-13

Updated Monitor docs to change more services to all services.

ktoliver committed:
Merge pull request #31467 from cwatson-cat/1-14-18NewButton-12

Updated Log Analytics docs to change more services to all services UI…

v-alje committed:
Merge pull request #31511 from MicrosoftDocs/master

1/16 AM Publish

rmca14 committed:
Merge pull request #31470 from cwatson-cat/1-14-18NewButton-15

Updated Backup docs to change more services to all services

rjagiewich committed:
Merge pull request #31472 from cwatson-cat/1-14-18NewButton-17

Updated Security topics to change more services to all services

cwatson-cat committed:
Merge branch 'release-portal-create-ui-change' of https://github.com/MicrosoftDocs/azure-docs-pr into 1-13-18NewButton-3

cwatson-cat committed:
Expanded ARM per guidelines

Commit:
acrolinx fixes

v-shils committed:
Merge pull request #31459 from cwatson-cat/1-13-18NewButton-3

Updated references from more services > all services

Commit:
fixed broken link

v-alje committed:
Merge pull request #31540 from MicrosoftDocs/master

1/16 PM Publish

ktoliver committed:
Merge pull request #31498 from craigcaseyMSFT/0115-craig-wb2

update files to change More Services to All Services

v-alje committed:
Merge pull request #31581 from MicrosoftDocs/master

1/17 AM Publish

v-shils committed:
Merge pull request #31083 from LizCasey/1-8-liz-wb

More changes for portal ui updates, task 1134169.

Commit:
change New to Create a resource for two files

v-alje committed:
Resolving merge conflicts

PRMerger19 committed:
Merge pull request #31600 from craigcaseyMSFT/0117-craig-wb

change New to Create a resource for two files

v-alje committed:
Merge pull request #31621 from MicrosoftDocs/master

1/17 PM Publish

v-alje committed:
Merge pull request #31668 from MicrosoftDocs/master

1/18 AM Publish

v-alje committed:
Resolving merge conflicts

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #31715 from MicrosoftDocs/master

1/18 PM Publish

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #31757 from MicrosoftDocs/master

1/19 AM Publish

Commit:
changed New to Create a resource and More services to All services

Commit:
acrolinx fixes

Commit:
acrolinx fixes

Commit:
update files to change More services to All services

PRMerger12 committed:
Merge pull request #31795 from craigcaseyMSFT/0119-craig-wb2

update files to change More services to All services

Commit:
updated files to change More services to All services

Commit:
Merge pull request #31797 from craigcaseyMSFT/0119-craig-wb3

updated files to change More services to All services

Commit:
updates to files to change More services to All services

v-alje committed:
Merge pull request #31803 from MicrosoftDocs/master

1/19 PM Publish

Commit:
updated one file and removed en-us from another

PRMerger17 committed:
Merge pull request #31802 from craigcaseyMSFT/0119-craig-wb4

updates to files to change More services to All services

Ja-Dunn committed:
Merge pull request #31783 from craigcaseyMSFT/0119-craig-wb1

changed New to Create a resource and More services to All services

Commit:
updated files to change More services to All services

v-alje committed:
Resolving merge conflicts

Commit:
Merge pull request #31806 from craigcaseyMSFT/0119-craig-wb5

updated files to change More services to All services

Commit:
updated files to change New to Create a resource

v-alje committed:
Merge pull request #31850 from MicrosoftDocs/master

1/22 AM Publish

Commit:
updated files to change New to Create a resource

v-shils committed:
Merge pull request #31849 from craigcaseyMSFT/1-22-craig-wb

updated files to change New to Create a resource

Commit:
updated files to change New to Create a resource

Ja-Dunn committed:
Merge pull request #31862 from craigcaseyMSFT/1-22-craig-wb2

updated files to change New to Create a resource

Ja-Dunn committed:
Merge pull request #31886 from craigcaseyMSFT/1-22-craig-wb3

updated files to change New to Create a resource

v-alje committed:
Merge pull request #31895 from MicrosoftDocs/master

1/22 PM Publish

Commit:
updated files to change More services to All services

v-alje committed:
Resolving merge conflicts

v-alje committed:
Bringing even with master.

Commit:
acrolinx fix

ktoliver committed:
Merge pull request #31903 from craigcaseyMSFT/1-22-craig-wb3

updated files to change More services to All services

Commit:
updated files to change More services to All services

v-alje committed:
Merge pull request #31953 from MicrosoftDocs/master

1/23 AM Publish

Lisaco88 committed:
Merge pull request #31950 from craigcaseyMSFT/1-23-craig-wb

updated files to change More services to All services

Commit:
updated files to change More services to All services

Ja-Dunn committed:
Merge pull request #31985 from craigcaseyMSFT/1-23-craig-wb2

updated files to change More services to All services

v-alje committed:
Merge pull request #31990 from MicrosoftDocs/master

1/23 PM Publish

v-alje committed:
Resolving merge conflicts

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #32039 from MicrosoftDocs/master

1/24 AM Publish

v-alje committed:
Merge pull request #32092 from MicrosoftDocs/master

1/24 PM Publish

v-alje committed:
Resolving merge conflicts

v-alje committed:
Merge branch 'release-portal-create-ui-change' of https://github.com/MicrosoftDocs/azure-docs-pr into release-portal-create-ui-change

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #32158 from MicrosoftDocs/master

1/5 AM Publish

v-alje committed:
Merge pull request #32192 from MicrosoftDocs/master

1/25 PM Publish

v-alje committed:
Resolving merge conflicts

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #32239 from MicrosoftDocs/master

1/26 AM Publish

v-alje committed:
Merge pull request #32277 from MicrosoftDocs/master

1/26 PM Publish

v-alje committed:
Resolving merge conflicts

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #32351 from MicrosoftDocs/master

1/29 AM Publish

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #32405 from MicrosoftDocs/master

1/29 AM Publish

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #32420 from MicrosoftDocs/master

1/29 PM release-event-grid-ga Publishing

v-alje committed:
Merge pull request #32476 from MicrosoftDocs/master

1/30 AM Publish

v-alje committed:
Merge pull request #32478 from MicrosoftDocs/master

1/30 AM release-mvc-batch Publish

v-alje committed:
Merge pull request #32521 from MicrosoftDocs/master

1/30 PM Publish

v-alje committed:
Resolving merge conflicts.

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #32576 from MicrosoftDocs/master

1/31 AM Publish

carolinacmoravia committed:
CC47408: Fix typo #429 [AUTOMATED]

Hello, @cwatson-cat, 

This proposed file change comes from this Community Contribution: https://github.com/MicrosoftDocs/azure-docs.ja-jp/pull/429/files

Please, help to check this proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

v-alje committed:
Merge pull request #32623 from MicrosoftDocs/master

1/31 PM Publish

v-alje committed:
Resolving merge conflict

v-alje committed:
Bringing even with master.

ellacroi committed:
updates on trial and ctonact notes

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into publishing-update

ellacroi committed:
fixing stylistic items in grow your business

ellacroi committed:
fixing request for trial in table

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into publishing-update

v-alje committed:
Merge pull request #32666 from MicrosoftDocs/master

2/1 AM Publish

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #32698 from MicrosoftDocs/master

2/1 PM Publish

v-nagta committed:
USER STORY 1176179 SaaS App Tutorial: Time management service

v-nagta committed:
updated

v-alje committed:
Merge pull request #32734 from MicrosoftDocs/master

2/2 AM Publish

v-alje committed:
Merge pull request #32767 from MicrosoftDocs/master

2/2 PM Publish

v-alje committed:
Resolving merge conflict

v-alje committed:
Bringing even with master.

v-nagta committed:
screenshots updated

v-nagta committed:
USER STORY 1177466 SaaS App Tutorial: Multi-tenant-application

v-nagta committed:
updatedemails

v-nagta committed:
updated

v-nagta committed:
URL pattern updated

v-alje committed:
Merge pull request #32830 from MicrosoftDocs/master

2/5 AM Publish

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #32874 from MicrosoftDocs/master

2/5 PM Publish

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #32917 from MicrosoftDocs/master

2/6 AM Publish

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #32953 from MicrosoftDocs/master

2/6 PM Publish

v-alje committed:
Bringing even with master.

v-alje committed:
Merge pull request #33004 from MicrosoftDocs/master

2/7 AM Publish

v-alje committed:
Merge pull request #33022 from MicrosoftDocs/master

2/7 PM Publish

v-alje committed:
Resolving merge conflicts

v-alje committed:
Merge branch 'release-portal-create-ui-change' of https://github.com/MicrosoftDocs/azure-docs-pr into release-portal-create-ui-change

v-alje committed:
Merge pull request #33052 from MicrosoftDocs/master

2/8 AM Publish

v-alje committed:
Merge pull request #33071 from MicrosoftDocs/master

2/8 PM Publish

v-alje committed:
Bringing even with master.

ellacroi committed:
version 1.1

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into publishing-update

carolinacmoravia committed:
C11939: Inconsistent format due to extra spaces 

Hello, @dsk-2015, 

Localization team has reported source content issue that causes localized version to have broken/different format compared to en-us version.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

v-alje committed:
Merge pull request #33120 from MicrosoftDocs/master

2/9 AM Publish

v-alje committed:
Merge pull request #33142 from MicrosoftDocs/master

2/9 PM Publish

v-nagta committed:
contentupdated

v-nagta committed:
USER STORY 1182476 SaaS App Tutorial: ZwayamSSO

v-nagta committed:
updated

v-alje committed:
Merge pull request #33201 from MicrosoftDocs/master

2/12 AM Publish

v-alje committed:
Resolving merge conflicts

v-alje committed:
Merge pull request #33232 from MicrosoftDocs/master

2/12 PM Publish

v-nagta committed:
USER STORY 1176179 SaaS App Tutorial: Attendance Management Services

v-nagta committed:
updated

v-nagta committed:
updatedurlpattern

v-nagta committed:
updated

billmath committed:
updating release notes

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into aadc

billmath committed:
updating

billmath committed:
updating

v-alje committed:
Merge pull request #33283 from MicrosoftDocs/master

2/13 AM Publish

v-alje committed:
Resolving merge conflicts

rothja committed:
Changing manager to craigg

v-alje committed:
Merge pull request #33332 from MicrosoftDocs/master

2/13 PM Publish

v-alje committed:
Bringing even with master.

v-nagta committed:
linkupdated

v-alje committed:
Merge pull request #33376 from MicrosoftDocs/master

2/14 AM Publish

v-alje committed:
Merge pull request #33413 from MicrosoftDocs/master

2/14 PM Publish

v-alje committed:
Resolving merge conflicts

cherylmc committed:
freshness

cherylmc committed:
format

cherylmc committed:
freshness

v-alje committed:
Bringing even with master.

cherylmc committed:
freshness

Commit:
Cosmos DB Emulator 1.20.108.4 release

This pull request updates the change list with what's in the Cosmos DB Emulator 1.20.108.4 release on February 14, 2018.

ellacroi committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into publishing-update

ellacroi committed:
fixing metadata

dominicbetts committed:
Split out RM FAQ questions

rothja committed:
Fixing broken relative links and some spelling

rothja committed:
Improving customer expectations of provisioning guide

dominicbetts committed:
Fix link

mumian committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

rothja committed:
Making titling and messaing on config guides and quickstarts more clear

v-alje committed:
Resolving merge conflicts

billmath committed:
updating

billmath committed:
updating status

Commit:
Documented our new PSModule

Section name: Controlling the Emulator. Also: added documentation for the new GetStatus option for CosmosDB.Emulator.exe.

v-alje committed:
Resolving merge conflict

mumian committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

MGoedtel committed:
updated to include querying of app insights app

Commit:
Updated ms.date

rothja committed:
Fixing Acrolinx warnings

MGoedtel committed:
updated sentence

MGoedtel committed:
Updated heading structure

rothja committed:
Fixing incorrect quickstart reference

Brenduns committed:
Update to metadata for authors and reviewers

Brenduns committed:
fixing last commit for accidental file inclusion

MGoedtel committed:
corrected first word in sentence

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into patch-2

v-geberr committed:
diff how entities are treated by LUIS

Commit:
Corrected a couple of spelling mistakes

markgalioto committed:
update the Azure index with new description for Backup service

MikeDodaro committed:
Add code segments to web search sdk Python.

MikeRayMSFT committed:
Staging additional step for DAG

gsacavdm committed:
Fixing gov-dns link

Undoing 69ca023

v-geberr committed:
cleanup

Commit:
Merge pull request #33502 from Brenduns/metadata-updates

Metadata updates

PRMerger10 committed:
Merge pull request #33506 from v-geberr/0215-entity-type

diff how entities are treated by LUIS

Brenduns committed:
Metadata changes only

MikeDodaro committed:
Revise per edit

Ja-Dunn committed:
Update Pre-builtEntities.md

date fix

Ja-Dunn committed:
Merge pull request #33507 from markgalioto/ab-2-15-2018

update the Azure index with new description for Backup service

Ja-Dunn committed:
Merge pull request #33514 from v-geberr/0215-howto-prebuilt-remove-comments

cleanup

GitHubber17 committed:
Merge pull request #33508 from MikeDodaro/addCodeWebPythonSDK

Add code segments to web search sdk Python.

klaaslanghout committed:
add specific error info when installing sync on DC

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fresh

v-alje committed:
Bringing even with master.

Commit:
Merge pull request #33513 from gsacavdm/patch-1

Fixing gov-dns link

ellacroi committed:
updating metadata on the grow your business article

Commit:
Merge pull request #33519 from cherylmc/fresh

Freshness review

shawntabrizi committed:
Updating RH docs with degraded status

shawntabrizi committed:
Update .gitignore

Ja-Dunn committed:
Merge pull request #32639 from ellacroi/publishing-update

Publishing update

shawntabrizi committed:
Acrolinx update

MGoedtel committed:
made edits to improve readability

shawntabrizi committed:
Update .gitignore

gunjanj committed:
Troubleshooting - Unable to update server endpoint 

Workaround to fix serverendpoint update issue

v-geberr committed:
added jp note

Commit:
Merge pull request #33520 from klaaslanghout/patch-13

add specific error info when installing sync on DC

GitHubber17 committed:
Merge pull request #33184 from v-nagta/zwayamsso

USER STORY 1182476 SaaS App Tutorial: ZwayamSSO

GitHubber17 committed:
Merge pull request #33492 from MGoedtel/UpdateLACrossWork215

updated to include querying of app insights app

GitHubber17 committed:
Merge pull request #33522 from shawntabrizi/master

Updating Resource Health Documentation for Degraded Status

v-nagta committed:
Date updated

dominicbetts committed:
Update metadata

ajaycode committed:
Minor changes.

Commit:
Merge pull request #33533 from ajaycode/patch-9

Minor changes.

v-nagta committed:
ZwayamSSO Icon Updated

v-geberr committed:
edits

Commit:
Revisions

Commit:
Update analysis-services-data-providers.md

v-geberr committed:
edits

Commit:
Merge pull request #33537 from Minewiskan/aasnuget

Revisions

mumian committed:
update HDI applications list and screenshots

mumian committed:
update the revision date

Commit:
Merge pull request #33523 from v-geberr/0215-supported-lang-jp-note

added jp note

PRMerger18 committed:
Merge pull request #33516 from Brenduns/metadata-updates

Metadata changes only

PRMerger19 committed:
Merge pull request #32587 from carolinacmoravia/patch-591

CC47408: Fix typo #429 [AUTOMATED]

Commit:
freshness

Commit:
removing an old note

Commit:
Merge pull request #33541 from Blackmist/freshness

freshness

gunjanj committed:
Updating Serverendpoint provisioning failed 

Minor edits

itechedit committed:
Merge pull request #33457 from dominicbetts/suite-split-faq

Split out RM FAQ questions

itechedit committed:
Merge pull request #33503 from David-Noble-at-work/patch-2

Cosmos DB Emulator 1.20.108.4 release

PRMerger15 committed:
Merge pull request #33524 from gunjanj/patch-7

Troubleshooting - Unable to update server endpoint

itechedit committed:
Merge pull request #33536 from v-nagta/missingiconv1

ZwayamSSO Icon Updated

GitHubber17 committed:
Merge pull request #33539 from mumian/master

[HDI] update HDI applications list and screenshots

rmca14 committed:
Merge pull request #33462 from rothja/provisioningcsat

Fixing manager metadata and improving CSAT of provisioning guide

anavinahar committed:
Update virtual-networks-faq.md

GitHubber17 committed:
Merge pull request #32808 from v-nagta/tenant

USER STORY 1177466 SaaS App Tutorial: Multi-tenant-application

PRMerger13 committed:
Merge pull request #33547 from anavinahar/patch-37

Update virtual-networks-faq.md

v-alje committed:
Resolving merge conflict

rmca14 committed:
Merge pull request #32721 from v-nagta/timeservice

USER STORY 1176179 SaaS App Tutorial: Time management service

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #33484 from MicrosoftDocs/release-portal-create-ui-change

Release portal create ui change

MikeRayMSFT committed:
Update per review from Curt Matthews.

v-alje committed:
Merge pull request #33550 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger14 committed:
Merge pull request #33279 from billmath/aadc

updating release notes

Commit:
Merge pull request #33119 from carolinacmoravia/patch-613

C11939: Inconsistent format due to extra spaces

Commit:
Merge pull request #33510 from MikeRayMSFT/20180215_LoadBalancerDAG

Staging additional step for DAG
